scene/gui/text_edit.cpp

```diff
@@ -2823,6 +2827,18 @@ bool TextEdit::is_editable() const {
 	return editable;
 }
 
+void TextEdit::set_fit_content_height(const bool p_enabled) {
+	if (fit_content_height == p_enabled) {
+		return;
+	}
+	fit_content_height = p_enabled;
+	update_minimum_size();
+}
+
+bool TextEdit::is_fit_content_height() const {
+	return fit_content_height;
+}
+
```

Comment:
`.h` and `.c` content should follow the same order, so this should be under `get_v_scroll_speed`, same with the other changes. Including moving `fit_content_height` next to `content_height_cache` in the header.